dev-src/dev/db.clj

```diff
@@ -18,8 +18,11 @@
   (db/transact seed-tx-data))
 
 (defn reset-db! []
-  (d/delete-database (config :datomic :uri))
-  (db/initialize)
+  ;; TODO: This isn't really correct. In Datomic you shouldn't delete and
```

Comment:
I'd be happy with a reminder about the kind of behavior we've seen in the past; this was related to the 5+ sec. response on the connection, or was it perhaps something else?